utility/pkgdb2.cfg.sample

```diff
@@ -92,3 +92,15 @@ PKGDB2_EMAIL_CC = None
 ## package or something alike. These emails are sent to the address set
 ## here:
 #MAIL_ADMIN = 'admin@pkgdb'
+
+
+## List the packages that are not accessible to the provenpackager group
+PKGS_NOT_PROVENPACKAGER = ['firefox']
```

Comment:
This is not the same as what the documentation says as default: 'firefox' vs 'firefox', 'thunderbird', 'xulrunner'